lib/commands/general.js

```diff
@@ -148,6 +148,29 @@ commands.getStrings = async function (language) {
   return this.apkStrings[language];
 };
 
+commands.startActivity = function (appPackage, appActivity) {
+  return this.adb.startApp({
+    pkg: appPackage,
+    activity: appActivity,
+    action: this.opts.intentAction,
+    category: this.opts.intentCategory,
+    flags: this.opts.intentFlags,
+    optionalIntentArguments: this.opts.optionalIntentArguments,
+    stopApp: this.opts.stopAppOnReset || !this.opts.dontStopAppOnReset,
+  });
+};
+
+commands.reset = async function () {
```

Comment:
https://github.com/appium/appium/blob/master/lib/appium.js#L380 we need this reset